src/core/ticker/Ticker.js

```diff
@@ -370,3 +457,42 @@ export default class Ticker
         this._maxElapsedMS = 1 / minFPMS;
     }
 }
+
+/**
+ * Lowest priority, useful for rendering.
+ * @static
+ * @constant
+ * @memberof PIXI.ticker.Ticker
+ * @type {number}
+ * @private
+ */
+Ticker.PRIORITY_LOW = 1;
+
+/**
+ * Medium priority, useful for post-interaction.
+ * @static
+ * @constant
+ * @memberof PIXI.ticker.Ticker
+ * @type {number}
+ * @private
+ */
+Ticker.PRIORITY_MEDIUM = 2;
+
+/**
+ * Highest priority, useful for interaction.
+ * @static
+ * @constant
+ * @memberof PIXI.ticker.Ticker
+ * @type {number}
+ * @private
+ */
+Ticker.PRIORITY_HIGH = 3;
+
+/**
+ * Default priority for adding ticker handlers.
+ * @static
+ * @constant
+ * @memberof PIXI.ticker.Ticker
+ * @type {number}
+ */
+Ticker.PRIORITY_DEFAULT = 10;
```

Comment:
What would you propose? Doesn't really matter to me, I was leaning on your previous suggestion for priority numbers ("The default priority could simply be 10, while internal PIXI consumers can add their listeners at specific priorities. The interaction manager at default - 3, the animated sprite default - 2, and the PIXI.Application render default - 1").